docs/2-agent-collaboration/1-events.md

```diff
@@ -38,7 +37,7 @@ EventHub.Publish(
 
 ## Receiving Events
 
-To receive events in a flow, subscribe to them in the flow's constructor using the `_eventHub.Subscribe` method:
+To receive events in a flow, subscribe to them in the flow's constructor using the `_messageHub.SubscribeAsyncChatHandler` method:
```

Comment:
This is not the chat handler. See an example below:             // Register handler for report generation events
            _messageHub.SubscribeFlowMessageHandler<ReportGenerationRequest>((args) =>
            {
                try
                {   
                    _eventQueue.Enqueue(args.Payload);
                    _logger.LogInformation($"Enqueued report generation event for URL {args.Payload.ProspectDetails.Url}");
                }
                catch (Exception ex)
                {
                    _logger.LogError($"Error processing event: {ex.Message}", ex);
                }
            });